pkg/common-controller/snapshot_controller.go

```diff
@@ -457,6 +456,9 @@ func (ctrl *csiSnapshotCommonController) syncUnreadySnapshot(snapshot *crdv1.Vol
 					ctrl.updateSnapshotErrorStatusWithEvent(snapshot, v1.EventTypeWarning, "SnapshotStatusUpdateFailed", fmt.Sprintf("Snapshot status update failed, %v", err))
 					return err
 				}
+
+				msg := fmt.Sprintf("waiting for a snapshot %s to be created by the CSI driver.", snapshot.Name)
+				ctrl.eventRecorder.Event(snapshot, v1.EventTypeNormal, "s/syncUnreadySnapshot/CreatingSnapshot", msg)
```

Comment:
By suggesting "s/syncUnreadySnapshot/CreatingSnapshot", I meant to do a string replacement - replace your original "syncUnreadySnapshot" with "CreatingSnapshot":).